src/pages/home.tsx

```
@@ -165,6 +204,51 @@ export default function HomePage() {
+      {/* Release Notes Dialog */}
+      <Dialog open={releaseNotesOpen} onOpenChange={setReleaseNotesOpen}>
+        <DialogContent className="max-w-4xl bg-(--docs-bg) pr-0 pt-4 pl-4 gap-1">
+          <DialogHeader>
+            <DialogTitle>What's new in v{appVersion}?</DialogTitle>
+            <Button
+              variant="ghost"
+              size="sm"
+              className="absolute right-10 top-2 focus-visible:ring-0 focus-visible:ring-offset-0"
+              onClick={() =>
+                window.open(
+                  releaseUrl.replace("?hideHeader=true&theme=" + theme, ""),
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)